BUILDING.md

```diff
@@ -19,13 +20,21 @@ Some versions of Android Studio ship with a built-in version of the Java JDK. Ho
 If you see any errors related to JDK version mismatch, download and install the correct version of the JDK (from the link above) and update your Android Studio settings to point to the correct JDK location:
 
 ```bash
-flutter config --jdk-dir /path/to/jdk
+fvm flutter config --jdk-dir /path/to/jdk
 ```
 
 ## Invoke Tasks
 
 We use the [invoke](https://www.pyinvoke.org) to run some core tasks - you will need python and invoke installed on your local system.
 
+## Flutter Version Management (FVM)
+
+This project uses [Flutter Version Management (FVM)](https://fvm.app/) to ensure consistent Flutter versions across development environments and CI/CD pipelines.
+
+For installation instructions, please refer to the [official FVM documentation](https://fvm.app/documentation/getting-started/installation).
+
+Once installed, FVM will automatically use the Flutter version specified in the `.fvmrc` file at the root of the project.
+
```

Comment:
Can you please add some notes here about:

- What commands to run to initially setup a local flutter SDK
- How to link to the SDK (e.g. in android studio)

Reply:
Added IDE setup instructions and a hint to run `fvm flutter doctor` as troubleshooting step.